addons/mail/static/src/composer/composer.xml

```diff
@@ -49,6 +49,7 @@
                             t-model="props.composer.textInputContent"
                             t-att-placeholder="placeholder"
                             t-att-readOnly="!state.active"
+                            t-att-disabled="thread and thread.type === 'chatter' and !thread.hasWriteAccess and !(thread.hasReadAccess and thread.canPostOnReadonly)"
```

Comment:
Again could also be abstract with getter `thread.isReadonly` or something like that, so that the conditions are simpler and it's more apparent it reuses the same concept as in other long boolean expressions of disabling some actions